targets/csharp/source/PlayFabSDK/source/Uunit/tests/PlayFabServerApiTest.cs

```diff
@@ -237,6 +240,60 @@ public void ParallelRequest(UUnitTestContext testContext)
                 }
             });
         }
+
+        /// <summary>
+        /// Multiplayer API
+        /// Try to deliberately request a mutliplayer alias that doesn't exist 
+        ///   Verify that response is MutliplayerServerNotFound, not just Not Found
+        /// </summary>
+        [UUnitTest]
+        public async void TestForNotFoundWithImportantInfo(UUnitTestContext testContext)
+        {
+            PlayFabSettings.staticSettings.TitleId = testTitleData.titleId;
+            PlayFabSettings.staticSettings.DeveloperSecretKey = testTitleData.developerSecretKey;
+
+            var eReq = new AuthenticationModels.GetEntityTokenRequest();
+            eReq.Entity = new AuthenticationModels.EntityKey();
+            eReq.Entity.Type = "title";
+            eReq.Entity.Id = testTitleData.titleId;
+
+            var tokenTask = await PlayFabAuthenticationAPI.GetEntityTokenAsync(eReq);
+
+            var staticPlayer = PlayFabSettings.staticPlayer;
+
+            if(tokenTask.Error != null)
+            {
+                testContext.Fail("Failed to retrieve the Title Entity Token, check your playFabSettings.staticPlayer, are they still logged in? (hint no server api should be called through a logged in client)");
+            }
+
+            if(aliasId == "")
```

Comment:
testContext.StringEquals(aliasId, "", "message");

Reply:
StringEquals is not what we want here. We would need to check for not equals. If these two strings match, we should fail. But how StringEquals works, if my valid aliasId is not "" than we end the test there (which we would only want this case to pass otherwise an empty alias would give us the incorrect information (bad parameter) that we expect back (multiplayer server not found)